.config/rollup.config.mjs

```diff
@@ -1,7 +1,6 @@
 import {nodeResolve} from '@rollup/plugin-node-resolve'; // so Rollup can resolve imports without file extensions and `node_modules`
 import babel from '@rollup/plugin-babel';
 import terser from '@rollup/plugin-terser';
-import pkg from '../package.json' assert { type: "json" };
```

Comment:
This wasn't compatible with Node 18, which is now part of the test matrix. It's possible that we don't care at all about multiple Node versions since this is all meant to run in the browser.